File: C1.UWP.FlexChart/CS/StockChart/StockChart/Converters/Color2BrushConverter.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using Windows.UI;
using Windows.UI.Xaml.Data;
using Windows.UI.Xaml.Media;

namespace StockChart.Converters
{
    public class Color2BrushConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            Color c = (Color)value;
            return new SolidColorBrush(c);
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            SolidColorBrush b = (SolidColorBrush)value;
            return b == null ? Colors.White: b.Color;
        }
    }
}

File: C1.UWP.FlexChart/VB/StockChart/StockChart/Converters/Color2BrushConverter.vb
Imports Windows.UI

Public Class Color2BrushConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Dim c As Color = DirectCast(value, Color)
        Return New SolidColorBrush(c)
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Dim b As SolidColorBrush = DirectCast(value, SolidColorBrush)
        Return If(b Is Nothing, Colors.White, b.Color)
    End Function
End Class